openwisp_monitoring/monitoring/base/models.py

```diff
@@ -962,7 +962,7 @@ def _is_crossed_by(self, current_value, time=None, retention_policy=None):
         if time is None:
             # retrieves latest measurements, ordered by most recent first
             points = self.metric.read(
-                since=f'{self._tolerance_search_range}m',
+                since=timezone.now() - timedelta(minutes=self._tolerance_search_range),
```

Comment:
Before this change, the code was iterating though points that was beyond the tolerance search range because of the broken InfluxQL query (`time >= 15m`).